packages/credit-card-integration/src/index.ts

```diff
@@ -0,0 +1,3 @@
+// export { default as createCreditCardPaymentStrategy} from "./lib/create-credit-card-payment-strategy";
+export { CreditCardPaymentInitializeOptions, WithCreditCardPaymentInitializeOptions } from "./lib/credit-card-payment-initialize-options";
```

Comment:
Why do we need `lib` sub-directory?

Reply:
`lib` comes from `nx` generator. I have removed the folder.